components/CommonCard/index.tsx

```diff
@@ -0,0 +1,77 @@
+import Card from '@mui/material/Card'
+import CardActions from '@mui/material/CardActions'
+import CardContent from '@mui/material/CardContent'
+import CardMedia from '@mui/material/CardMedia'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Chip from '@mui/material/Chip'
+
+const styledCard = {
+  width: '400px',
+  height: '500px',
+  background: '#FFFFFF',
+  boxShadow: '0px 0px 0px 1px #E0E0E0',
+  borderRadius: '25px',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  margin: 1,
+}
+
+const styledCardContent = {
+  display: 'flex',
```

Comment:
I think we can add a `height: "100%"` here so that the content will move to fill in the empty space if there are no actions passed to the CommonCard i.e No buttons needed for the CommonCard. 

However, the alignment might look weird if we have 2 cards side by side, where 1 has a button & the other does not.

Current implementation:
![Screenshot 2023-04-12 at 3 44 31 PM](https://user-images.githubusercontent.com/90279138/231388275-9effee46-b1af-4f83-973d-53fb75ca012c.png)

With `height: "100%"`:
![Screenshot 2023-04-12 at 3 44 22 PM](https://user-images.githubusercontent.com/90279138/231388438-30447068-fbfb-4395-ad03-58b309f6ed39.png)

Reply:
Thanks for raising this up. I have updated the height to not be a fixed number, so now there will not be any empty spaces if the card does not have actions and is put beside cards without actions.